pkgs/tools/filesystems/dwarfs/default.nix

```
@@ -0,0 +1,104 @@
+{ stdenv
+, substituteAll
+, fetchFromGitHub
+, lib
+
+, bison
+, boost
+, cmake
+, double-conversion
+, fmt_8
+, fuse3
+, gflags
+, glog
+, gtest
+, jemalloc
+, libarchive
+, libevent
+, libunwind
+, lz4
+, openssl
+, pkg-config
+, ronn
+, xxHash
+, xz
+, zstd
+}:
```

You forgot to remove `xz` from here. Plus, we prefer having `lib` first.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, substituteAll
, bison
, boost
, cmake
, double-conversion
, fmt_8
, fuse3
, gflags
, glog
, gtest
, jemalloc
, libarchive
, libevent
, libunwind
, lz4
, openssl
, pkg-config
, ronn
, xxHash
, zstd
}:
```